src/apps/content-editor/src/app/views/ItemEdit/components/Header/DuoModeToggle/DuoModeToggle.js

```diff
@@ -52,23 +53,29 @@ export function DuoModeToggle(props) {
           </Fragment>
         )
       ) : (
-        <ToggleButton
+        <ToggleButtonGroup
           title="Duo Mode Toggle"
-          className={styles.ToggleButton}
-          name={props.name}
+          color="secondary"
           value={Number(ui.duoMode)}
-          offValue={<FontAwesomeIcon icon={faWindowClose} />}
-          onValue={<FontAwesomeIcon icon={faDesktop} />}
-          onChange={(val) => {
-            if (val == 1) {
+          exclusive
+          size="small"
+          onChange={(e, val) => {
+            if (val === 1) {
               dispatch(actions.setDuoMode(true));
               dispatch(actions.setContentActions(false));
-            } else {
+            } else if (val === 0) {
               dispatch(actions.setDuoMode(false));
               dispatch(actions.setContentActions(true));
             }
           }}
-        />
+        >
+          <ToggleButton value={0}>
+            <CloseIcon fontSize="small" />
+          </ToggleButton>
+          <ToggleButton value={1}>
```

Comment:
@shrunyan we are setting the Numeric ToggleButton value here to answer your above question.